packages/apps/spaces/components/contact/contact-history/ContactHistory.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useContactNotes } from '../../../hooks/useContactNote/useContactNotes';
+import { Timeline } from '../../ui-kit/organisms';
+import { useContactConversations } from '../../../hooks/useContactConversations';
+import { useContactActions } from '../../../hooks/useContactActions';
+
+export const ContactHistory = ({ id }: { id: string }) => {
+  const {
+    data: notes,
+    loading: notesLoading,
+    error: notesError,
+  } = useContactNotes({ id });
+  // const {data: actions, loading: actionsLoading, error: actionsError} = useContactActions({id});
+  // TODO add pagination support
+  const {
+    data: conversations,
+    loading: conversationsLoading,
+    error: conversationsError,
+  } = useContactConversations({ id });
```

Comment:
[neat pick] those calls could be moved to single hook that returns already sorted items
 (you can ignore it)

Reply:
i'll wrap this later when i add back actions